tests/Notes.org

```diff
@@ -403,6 +403,7 @@ i
 * MobyDick
   :PROPERTIES:
   :NOTER_DOCUMENT: MobyDick.pdf
+  :NOTER_PAGE: 171
```

Comment:
Should we introduce a new property like `LAST_READ_PAGE`? Or has this one been used historically?

Reply:
`NOTER_PAGE` is historical -- coming to us from GC.  I don't think the double usage of the property is problematic, since the context is very different.